lib/rust/ensogl/component/list-view/src/entry.rs

```diff
@@ -83,13 +87,14 @@ pub trait Entry: CloneRef + Debug + display::Object + 'static {
 // === Label ===
 
 /// The [`Entry`] being a single text field displaying String.
+#[allow(missing_docs)]
 #[derive(Clone, CloneRef, Debug)]
 pub struct Label {
     display_object: display::object::Instance,
-    label:          text::Area,
+    pub label:      text::Area,
     text:           Rc<RefCell<String>>,
     max_width_px:   Rc<Cell<f32>>,
-    network:        enso_frp::Network,
+    pub network:    enso_frp::Network,
```

Comment:
I'm not sure you want to make it public. Why is it needed?

Reply:
As we discussed on Discord together with @farmaazon:
 - we extend this network in `component_group::entry::Entry`;
 - I added a comment for the `network` field, mentioning caveats to extending it.

Reply:
As we discussed, please add a comment with detailed explanation of the use case and why it's needed. Comment `Note: Networks extending this field will not outlive [`Label`].` is not enough. I would like to see here explanation WHY this field is made public and why is it needed.